test/statistics.e2e-spec.ts

```
@@ -0,0 +1,74 @@
+import { Test, TestingModule } from '@nestjs/testing';
+import { INestApplication } from '@nestjs/common';
+import request from 'supertest';
+import { AppModule } from './../src/app.module';
+
+jest.setTimeout(30000);
```

ESLint

[error] Invalid group length in numeric value.

(unicorn/numeric-separators-style)